wrappers/python/nntile/layer/llama_attention.py

```diff
@@ -1517,3 +1517,169 @@ def to_torch_with_grads(self) -> LlamaAttention_torch:
                 to_numpy(self.in_proj_bias_v.grad).T.flatten()
             )
         return torch_layer
+
+    def get_layer_forward_flops(self):
+        total_forward_flops = 0
+        # Compute Q_transposed
+        # gemm_async(
+        #     1.0,
+        #     notrans,
+        #     self.w_q.value,
+        #     notrans,
+        #     self.x_q.value,
+        #     0.0,
+        #     self.q_transposed.value,
+        #     1,
+        #     0,
+        #     redux=self.redux,
+        # )
+        w_q_shape = self.w_q.value.shape
+        x_q_shape = self.x_q.value.shape
+        qt_flops = np.prod(w_q_shape) * np.prod(x_q_shape[1:])
+        total_forward_flops += qt_flops
+        # Compute K_transposed
+        # gemm_async(
+        #     1.0,
+        #     notrans,
+        #     self.w_k.value,
+        #     notrans,
+        #     self.x_k.value,
+        #     0.0,
+        #     self.k_transposed.value,
+        #     1,
+        #     0,
+        #     redux=self.redux,
+        # )
+        w_k_shape = self.w_k.value.shape
+        x_k_shape = self.x_k.value.shape
+        kt_flops = np.prod(w_k_shape) * np.prod(x_k_shape[1:])
+        total_forward_flops += kt_flops
+        # Compute V_transposed
+        # gemm_async(
+        #     1.0,
+        #     notrans,
+        #     self.w_v.value,
+        #     notrans,
+        #     self.x_v.value,
+        #     0.0,
+        #     self.v_transposed.value,
+        #     1,
+        #     0,
+        #     redux=self.redux,
+        # )
+        w_v_shape = self.w_v.value.shape
+        x_v_shape = self.x_v.value.shape
+        vt_flops = np.prod(w_v_shape) * np.prod(x_v_shape[1:])
+        total_forward_flops += vt_flops
+        # Compute tensor for softmax
+        # gemm_async(
+        #     1.0 / self.head_size**0.5,
+        #     trans,
+        #     self.k_rep.value,
+        #     notrans,
+        #     self.q_rope.value,
+        #     0.0,
+        #     self.a.value,
+        #     1,
+        #     3,
+        #     redux=self.redux,
+        # )
+        k_rep_shape = self.k_rep.value.shape
+        q_rope_shape = self.q_rope.value.shape
+        a_flops = np.prod(k_rep_shape[:2]) * np.prod(q_rope_shape[1:])
```

Comment:
`gemm_ndim=1` and `batch_ndim=3` and `k` is transposed. Seems like FLOPS count is incorrect. Correct value seems to be `2*np.prod(k_rep_shape)*q_rope_shape[0]`

Reply:
```
K: (N1, N2, N3, N4, N5)
Q: (N6, N1, N3, N4, N5)

Flops=2 * (N1*N2*N3*N4*N5) * N1
```

Reply:
Why does ``N6`` not affect the flops?

Reply:
because I made a mistake. It shall be `Flops = 2 * (N1*N2*N3*N4*N5) * N6`

Reply:
And `Q: (N1, N6, N3, N4, N5)`

Reply:
Then, my code worked correctly (up to factor 2):
1) ``np.prod(k_rep_shape[:2]) = N1 * N2``
2) ``np.prod(q_rope_shape[1:]) = N6 * N3 * N4 * N5``

Reply:
indeed